### PR TITLE
Lavaland seed vault and ash walker ruin remap

### PR DESCRIPTION
**Pull request by WJohn**
This remaps the lavaland seed vault ruin, ash walker nest, and adds two new nests.

:cl: WJohnston
tweak: Remapped the lavaland seed vault ruin.
tweak: Remapped the ashwalker nest and added two new variations.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
WJ its supposed to be a space ship

**Comment by Hatfish** [comment]
Any chance you could toss two more hydroponics trays in there so each plantman has two to themselves?

**Comment by WJohn** [comment]
Sure thing, I'll also change the flavor text for their spawners.

**Comment by Armhulen** [comment]
>sent from the cosmos to save other races

>built in turrets

I mean, call me picky..

**Comment by KorPhaeron** [comment]
This is a nicer map but it violates the DEEP LORE while also making it functionally less appealing to play in (shittier botany setup)

**Comment by WJohn** [comment]
I changed the flavor text, like I said. It's now something about a research experiment gone haywire.

**Comment by KorPhaeron** [comment]
Obviously I'm biased but I think space plant people bearing gifts is more exciting I guess we'll let the dumb emoji feedback decide

**Comment by KorPhaeron** [comment]
@bobdobbington 

Botany expert please tell me which botany is the best

**Comment by Armhulen** [comment]
did someone say dumb emoji feedback?
👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌there👌👌👌 right✔there ✔✔if i do ƽaү so my self 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠOOOOOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shit

**Comment by Iamgoofball** [comment]
sign me the FUCK up 👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌th 👌 ere👌👌👌 right✔there ✔✔if i do ƽaү so my selｆ 💯 i say so 💯 thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shit

**Comment by MMMiracles** [comment]
do NOT sign me the FUCK up :-1::eyes::-1::eyes::-1::eyes::-1::eyes::-1::eyes: bad shit ba̷̶ ԁ sHit :-1: thats :x: some bad :-1::-1:shit right :-1::-1: th :-1: ere :-1::-1::-1: right :x: there :x: :x: if i do ƽaү so my selｆ:no_entry_sign: i say so :no_entry_sign: thats not what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ :no_entry_sign: :-1: :-1::-1:НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ :-1: :-1::-1: :-1: :no_entry_sign: :-1: :eyes: :eyes: :eyes: :-1::-1:Bad shit

Seriously though plant people being sent across the galaxy in hopes of spreading the gift of nature is a lot cooler than hippies who got overrun by their plants.

**Comment by WJohn** [comment]
they're both hippies though

besides, they're both going to be spreading their gift

these ones are just going to be much more aggressive about it.

**Comment by KorPhaeron** [comment]
flavour_text = "<font size=3><b>A</b></font><b>n eerie warning siren is heard as you awaken from your vine-covered pod. You slowly remember what happened. Research... a breakthrough... harmony... unity... panic? None of that matters now. You no longer fear what you have created, and are glad it has grown so prosperously in your absence. There is still work to be done. See to it that you spread your discoveries to others.</b>";

**Comment by KorPhaeron** [comment]
For those who dont want to look in the file

**Comment by Armhulen** [comment]
>preserved terrarium
>vine covered pod
pick one dammit

**Comment by WJohn** [comment]
I already renamed the pod to an "ancient sleeper" or something like that.

**Comment by coiax** [comment]
@WJohn while you are changing the seed vault, make sure to change the name and description in https://github.com/tgstation/tgstation/blob/ab97b0b320e2d19497144861e8ef0420e7545492/code/datums/ruins/lavaland.dm#L51 and set allow_duplicates to FALSE

**Comment by ike709** [comment]
do NOT sign me the FUCK up :-1::eyes::-1::eyes::-1::eyes::-1::eyes::-1::eyes: bad shit ba̷̶ ԁ sHit :-1: thats :x: some bad :-1::-1:shit right :-1::-1: th :-1: ere :-1::-1::-1: right :x: there :x: :x: if i do ƽaү so my selｆ:no_entry_sign: i say so :no_entry_sign: thats not what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ :no_entry_sign: :-1: :-1::-1:НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ :-1: :-1::-1: :-1: :no_entry_sign: :-1: :eyes: :eyes: :eyes: :-1::-1:Bad shit

I'm just doing what all the cool kids are doing. Lavaland wasn't even a thing the last time I played SS13. So :+1: / :-1:

**Comment by Armhulen** [comment]
I'd be okay if they were plant people who rose up and killed their creators and are now chill as fuck 👌

**Comment by KorPhaeron** [comment]
S P A C E S H I P

**Comment by WJohn** [comment]
Is this PR seriously being held up because of some tiny lore tidbit that has already been changed? Who cares if it's a spaceship or not?

**Comment by Bawhoppen** [comment]
I warned you WJ

**Comment by Cheridan** [comment]
DEEPLORE aside, this seems like a lot of wasted space around a relatively tiny botany setup..

**Comment by AlexCrimson** [comment]
Have you seen the size of some of the Lavaland ruins? This thing is tiny compared to some. Just stick some fucking thrusters on the back of the damn thing if its an issue. Not like its a rule to make all spaceships symmetrical.

**Comment by coiax** [comment]
If it doesn't have tiny fans at the exit airlock, it needs some sort of scrubber system, because otherwise the air will become unbreathable thanks to planet atmos.

**Comment by WJohn** [comment]
Ah, good catch. I'll add in a fan.

**Comment by WJohn** [comment]
I am a colossal idiot and accidentally did my changes from my other branch onto this one.

OH
WELL

**Comment by KorPhaeron** [comment]
>tripling ash walker spawn rate

if this PR wasn't in merge hell before it sure is now

**Comment by WJohn** [comment]
i like ash walkers tho

**Comment by AlexCrimson** [comment]
Should probably add that in a different PR

**Comment by WJohn** [comment]
Yeah I was going to and then I goofed it up and accidentally put it in this one and I have no idea how to git to undo any of this mess.

File: _maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm [hunk 1]
```diff
@@ -1,582 +1,1196 @@
...
 	},
-/area/ruin/unpowered)
+/turf/open/floor/plating/asteroid/basalt/lava_land_surface,
+/area/awaycontent/a2{
+	has_gravity = 1;
+	name = "Ashwalker Nest 1"
+	})
 "m" = (
-/obj/structure/closet/crate/medical,
-/obj/item/device/malf_upgrade,
```
Comment (GunHog) [inline _maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm]: Why did you remove this?

**Comment by WJohn** [comment]
I removed most of the high-tech stuff that they have, I know ashwalkers have scavenged a lot of stuff already, but I'd like it if they had less technologically advanced stuff to work with overall.

**Comment by optimumtact** [comment]
Not only is it being held up, but if you modify the PR to confirm to Kor's wishes it will be closed as well.

**Comment by optimumtact** [comment]
Oh by the way I assume the ash ruins are fine, it's just the spawn rate and the seedvault ruin not being a spaceship anymore that are holding this back.

**Comment by GunHog** [comment]
> I removed most of the high-tech stuff that they have, I know ashwalkers have scavenged a lot of stuff already, but I'd like it if they had less technologically advanced stuff to work with overall.

Ash Walkers have zero use for the Malf Disk, WJ. It is the only way the station can get one, and it was one of the few rewards for assaulting one of the biggest threat to miner players short of boss monsters: Hostile players which are the Lavaland equivalent of spear-assistants. Your point is not valid for the justification of removing reward items.

**Comment by optimumtact** [comment]
What, why on earth do ashwalkers have a malf disk, that's dumb

**Comment by coiax** [comment]
Someone please guide WJohn through undoing the merge of the other branch #atomicprs

File: code/datums/ruins/lavaland.dm [hunk 2]
```diff
@@ -51,22 +51,45 @@
...
+	id = "ash-walker2"
+	description = "A race of unbreathing lizards live here, that run faster \
+		than a human can, worship a broken dead city, and are capable of \
+		reproducing by something involving tentacles? Probably best to \
+		stay clear."
+	suffix = "lavaland_surface_ash_walker2.dmm"
+	cost = 15
+	allow_duplicates = FALSE
+
+/datum/map_template/ruin/lavaland/ash_walker
```
Comment (ChangelingRain) [inline code/datums/ruins/lavaland.dm]: Okay, so there's kind of an issue with how you set this up; as-is, these are still counted as one type, and only one of the three would actually show up, you want to do something like this:

    /datum/map_template/ruin/lavaland/ash_walker
    	name = "Ash Walker Nest Layout 1"
    	id = "ash-walker"
    	description = "A race of unbreathing lizards live here, that run faster \
    	than a human can, worship a broken dead city, and are capable of \
    	reproducing by something involving tentacles? Probably best to \
    	stay clear."
    	suffix = "lavaland_surface_ash_walker1.dmm"
    	cost = 20
    	allow_duplicates = FALSE

    /datum/map_template/ruin/lavaland/ash_walker/altone
    	name = "Ash Walker Nest Layout 2"
    	suffix = "lavaland_surface_ash_walker2.dmm"

    /datum/map_template/ruin/lavaland/ash_walker/alttwo
    	name = "Ash Walker Nest Layout 3"
    	suffix = "lavaland_surface_ash_walker3.dmm"
    

That, if copied in and tabbed properly, will have three layouts for one ruin; it'll pick one of the three, and only one.

**Comment by jughu** [comment]
This only be cool if i can put the miners in plant pods and turn them into one of us

**Comment by KorPhaeron** [comment]
The new nests seem a bit sterile/organized as well.

**Comment by Armhulen** [comment]
this is by far the most spaghetti pr i have ever seen. I like the nests but three times the spawnrate is pretty scary, don't know how i feel about that

**Comment by WJohn** [comment]
It's not three times spawn rate anymore, only one of those three will be chosen.

**Comment by KorPhaeron** [comment]
S P A C E S H I P
A
C
E
S
H
I
P

**Comment by GunHog** [comment]
I would like this change if you had not removed the malf disk. It is a nice and hyper rare (due to the fact that you have to fight a group of enemy players to get) reward for the massive risk involved in assaulting Ash Walkers.

On your maps, I do not see anything a miner would really care to take back home with him, only the minerals and monster drops the Ash Walkers tend to collect. Of course, the more loot they have, the more dangerous they are.

**Comment by AlexCrimson** [comment]
What does the malf disk even do? I have never seen a miner grab it let alone use it for anything. Is it really a big deal?

**Comment by GunHog** [comment]
It is just that WJ has not provided a good reason to remove it, Ash Walkers cannot even use it. The Malf Disk, as the name implies, Malfs an AI. It gives the AI Traitor/Malfunction powers without affecting its laws or making it an antagonist. Most miners ignore it, not knowing what it is or does. On the 3-ish rounds that I was Malf'ed using the disk, I have found it fun, and I think it a shame to lose that.

It is such a rare occurrence that oranges did not even know Ash Walker nests have them. At the very least, I suggest replacing the reward with something else. Right now, WJ's map does not provide any reward for having the stones to assault an Ash Walker base (other than stopping them being a threat). I think that is a real shame.

The reward does not **have** to be the malf disk, but should be something of high value to at least one player on the station.

**Comment by KorPhaeron** [comment]
Okay here is my FULL LIST

-I prefer the space ship lore. MUH OPINIONS

-The botany space is shittier now function wise. The current botany space was after ~3 PRs updating from player feedback and observing how they arranged the room set ups over many rounds. 

-I'm still not sure simulating power is worth it. They have no way to replenish the vault, unless the SMES is infinite, in which case why have a SMES at all?

-Less plantmen spawns? Why?

-Ash lizard huts are disconnected/tiny. This is bad for being able to do stuff indoors during storms (unless the tunnels are stormproof, but then that gets even weird)

-No malf disk?

-I preferred the cluttered shitheap of random stolen stuff. It looks neatly organized now.

-They start with bone gear/bones. Why? It's supposed to be a hunting reward. They have little enough to do as is.

I guess the layouts of both are sorta growing on me aesthetically but that is my list of dislikes.

I feel like this PR is probably gonna drag on until there is a compromise that neither of us really like but I like you and your mapping so I don't want to close it outright and here we are

**Comment by WJohn** [comment]
-I can definitely see about improving the lore and botany's size, it can be made a lot bigger pretty easily. 
-The SMES cell can be powered by the generator found right next to it, a space suit and pick is provided to them if they need to make some plasma. It should also last all round anyway.
-There's the same 4 podpeople spawns as last time, nothing's changed there.
-I wanted to change up how the lizard's nests were designed to be built into tunnels like a small community, and these tunnels are shielded from the storms.
-I can re-add the malf disk.
-It's actually quite cleanly organized in the current lizard nest, too. It becomes a cluttered shitheap because of the players.
-Each of the 3 nests has slightly different starting gear. The first one starts with a tamed goliath that will fight lavaland mobs for them, the second starts with a full set of bone gear armor but one less egg (likely for the leader), and the third has much better medical equipment and an extra toolbox.

**Comment by Hatfish** [comment]
Rusty walls are the ugliest shit, is there any chance that you could replace them with rough metal walls or something?

**Comment by WJohn** [comment]
I actually kinda like our rusty walls... Heck I want to make walls that are even rustier than that.

There's not much option for walls otherwise, I could do stuff like iron mineral walls which look like silver/gold mineral walls but grey and that's about it.

**Comment by KorPhaeron** [comment]
Please atomize the PR when you're ready to work on it again.

Or if you figure out how to move the lizard stuff into another branch I'll reopen this one
